test/e2e/logforwarding/multiple_outputs/forward_to_multiple_outputs_test.go

```diff
@@ -139,13 +139,13 @@ func newClusterLogForwarder(fluentRcv *apps.Deployment, elasticsearch *eologging
 	fluentdOutput := loggingv1.OutputSpec{
 		Name: fluentRcv.GetName(),
 		Type: loggingv1.OutputTypeFluentdForward,
-		URL:  fmt.Sprintf("%s.%s.svc:24224", fluentRcv.GetName(), fluentRcv.GetNamespace()),
+		URL:  fmt.Sprintf("tcp://%s.%s.svc:24224", fluentRcv.GetName(), fluentRcv.GetNamespace()),
 	}
 
 	elasticOutput := loggingv1.OutputSpec{
 		Name: elasticsearch.GetName(),
 		Type: loggingv1.OutputTypeElasticsearch,
-		URL:  fmt.Sprintf("%s.%s.svc:9200", elasticsearch.GetName(), elasticsearch.GetNamespace()),
+		URL:  fmt.Sprintf("https://%s.%s.svc:9200", elasticsearch.GetName(), elasticsearch.GetNamespace()),
```

Comment:
`https` depends if `pipelineSecret` is set.

Reply:
Fixed